### PR TITLE
Naive baseline processor picture

### LINKED ISSUE #10: Implement naive Baseline image Processor
Prerequisites:

- [x] #8

Todo:

Implement the existing example with Gaussian noise as an Processor

### PR DESCRIPTION
**Pull request by EhrenbruderRoschtbraten**
Implemented the Naive Baseline Processor for images based on the fake generator path, so this should only be merged if the fake generator branch is merged.

closes #10

### CODE REVIEW

File: src/fool_ai_detector/service/naive_baseline_processor_picture.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        """
+        Adds some random noise to an image to augment it.
+        """
+        # read image file,
+        image = skimage.io.imread(input_file_path)
+        # augment the image
+        processed_image = skimage.util.random_noise(image, mode='gaussian', clip=True)
+        # convert pixel representation from [0..1] to [0..255]
+        processed_image = (processed_image * 255).astype(np.uint8)
+        # save image to given location
```
Comment (1Maxnet1) [inline src/fool_ai_detector/service/naive_baseline_processor_picture.py:19]: Could we either remove the inline comments or shift them to the pydoc comment?

Comment (EhrenbruderRoschtbraten) [reply src/fool_ai_detector/service/naive_baseline_processor_picture.py]: are removed

**Review by 1Maxnet1 — APPROVED** [review]
LGTM

File: src/fool_ai_detector/service/naive_baseline_processor_picture.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class NaiveBaselineProcessorImage(Processor):
+    def process(self, input_file_path, output_file_path):
+        """
+        Adds some random noise to an image to augment it.
+        """
+        image = skimage.io.imread(input_file_path)
+        processed_image = skimage.util.random_noise(image, mode='gaussian', clip=True)
+        processed_image = (processed_image * 255).astype(np.uint8)
+        skimage.io.imsave(output_file_path, processed_image, check_contrast=False)
```
Comment (RaphaW) [inline src/fool_ai_detector/service/naive_baseline_processor_picture.py:16]: Could you add a new line here please

File: src/fool_ai_detector/service/naive_baseline_processor_picture.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import os
+import numpy as np
+import skimage
```
Comment (1Maxnet1) [inline src/fool_ai_detector/service/naive_baseline_processor_picture.py:3]: Could you please add skikit-image to the pyproject.toml as a dependency?

**Review by 1Maxnet1 — CHANGES_REQUESTED** [review]
One last thing, then it is goo :)

**Review by LeviUniAcc — APPROVED** [review]
ty for changeing test :D

**Review by 1Maxnet1 — APPROVED** [review]
LGTM, strict versions are impractical in my experience, because if another package we use needs another scikit-image version, it will fail to install. However we can still change this when necessary.

**Comment by EhrenbruderRoschtbraten** [comment]
> LGTM, strict versions are impractical in my experience, because if another package we use needs another scikit-image version, it will fail to install. However we can still change this when necessary.

Ah no I'll change it here.

**Comment by EhrenbruderRoschtbraten** [comment]
We can then merge it in the discussion later

**Review by RaphaW — APPROVED** [review]
LGTM

**Pull request merged**